gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/model/GbUser.java

```diff
@@ -91,15 +91,24 @@ public boolean isValid() {
 		return StringUtils.isNotBlank(userUuid);
 	}
 
+	public String getSortName() {
+		StringBuilder sb = new StringBuilder(128);
+		if(lastName != null) {
+			sb.append(lastName);
+		}
+		if (firstName != null) {
+			if (lastName != null) {
+				sb.append(", ");
+			}
+			sb.append(firstName);
+		}
+		return sb.toString();
+	}
+
 	@Override
 	public int compareTo(GbUser user)
 	{
-		int comp = displayId.compareToIgnoreCase(user.displayId);
-		if (comp == 0) {
-			comp = displayName.compareToIgnoreCase(user.displayName);
-		}
-
-		return comp;
+		return getSortName().compareTo(user.getSortName());
```

Comment:
Any reason why you are making this case sensitive now? In my experience users prefer case-insensitive sorting for names.

Reply:
No, just missed, I corrected and now use compareToIgnoreCase.